### PR TITLE
[ADD] newsletter_save_recipients

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

File: newsletter_save_addressee/model/newsletter_newsletter.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            'res.partner',
+            'newsletter_partner_rel', 'newsletter_id', 'partner_id',
+            string='Partners'),
+        'type_model': fields.related(
+            'type_id', 'model', 'model', type='char', string='Model'),
+    }
+
+    def _do_send_newsletter(self, cr, uid, this, record_id, context=None):
+        result = super(NewsletterNewsletter, self)._do_send_newsletter(
+            cr, uid, this, record_id, context=None)
```
Comment (guewen) [inline newsletter_save_addressee/model/newsletter_newsletter.py]: should be `context=context`

**Comment by guewen** [comment]
Apart the typo noticed above, seems good to me. Travis is not happy but because of another module.

**Comment by hbrunn** [comment]
Thanks, I also found a crucial other problem...

**Comment by guewen** [comment]
:+1:

**Comment by StefanRijnhart** [comment]
Nifty trick, exposing the many2many table as its own model. Just out of curiosity, was there any particular reason for that?

But really, :+1:

**Comment by max3903** [comment]
The pot file is missing

:-1:

File: newsletter_save_addressee/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Save addressees of newsletters",
```
Comment (pedrobaeza) [inline newsletter_save_addressee/__openerp__.py:22]: Typo: s/addressees/addresses

**Comment by pedrobaeza** [comment]
The typo error is also in the module name.

File: newsletter_save_addressee/__openerp__.py (hunk 2 above)
Comment (bwrsandman) [inline newsletter_save_addressee/__openerp__.py:22]: I don't think it's meant to be address
https://www.wordnik.com/words/addressee

Comment (pedrobaeza) [inline newsletter_save_addressee/__openerp__.py:22]: OK, I understand. I didn't know about that term.

**Comment by pedrobaeza** [comment]
Forget about my last comments. It's a lack of vocabulary. :+1:

**Comment by bwrsandman** [comment]
@pedrobaeza go easy on the merge-happiness, did you not see @max3903's comment https://github.com/OCA/crm/pull/6#issuecomment-62200686

There is one +1 (excluding yours) and one -1, this should not have been merged

**Comment by pedrobaeza** [comment]
Sorry, I forgot it completely!, because I think the only concerns about this module were mine. I will add it via direct commit.

**Comment by bwrsandman** [comment]
@pedrobaeza Thank you

**Comment by pedrobaeza** [comment]
Done on https://github.com/OCA/crm/commit/4620d776bdc23b577aaa41824aa733dfeefa39ea

**Pull request merged**